server/src/internalClusterTest/java/org/opensearch/remotestore/CreateRemoteIndexIT.java

```diff
@@ -105,6 +127,26 @@ public void testDefaultRemoteStoreNoUserOverride() throws Exception {
         );
     }
 
+    private static final String SYSTEM_INDEX_NAME = ".test-system-index";
+
+    public void testSystemIndexWithRemoteStoreClusterSetting() throws Exception {
+        IllegalArgumentException illegalArgumentException = expectThrows(
```

Comment:
What are the settings used in this createIndex call?

Reply:
As we didn't pass any index settings it will use the parent class `OpenSearchIntegTestCase` index settings (which randomly picks no.of shards and replicas). For node settings we are overriding node settings at top of the class, so we will use those node settings

Reply:
Okay, I see this is removed now so resolving this conversation.

Reply:
Sorry for the confusion, had to add it back again